crates/libs/bindgen/src/metadata.rs

```diff
@@ -240,7 +240,7 @@ impl Signature {
         }
 
         // Then we see if we can infer retval-like behavior more conservatively.
-        self.params.last().map_or(false, |param| param.is_retval())
+        self.params.last().is_some_and(|param| param.is_retval())
```

Comment:
Clippy?

Reply:
Yep, can pull this out if you'd prefer to address in a standalone PR.